importer/Uploader.py

```diff
@@ -46,11 +46,12 @@ def add_descriptions(self, target_item, descriptions, log):
             desc_content = descriptions[description]['value']
             lang = descriptions[description]['language']
             self.wdstuff.add_description(
-                lang, desc_content, target_item, summary=self.summary)
+                lang, desc_content, target_item)
             if log:
                 t_id = target_item.getID()
-                log.logit(t_id + " ADDED DESCRIPTION " +
-                          lang + " " + desc_content)
+                message = "{} ADDED DESCRIPTION {} {}".format(
+                    t_id, lang, desc_content)
+                log.logit(message)
 
     def add_aliases(self, target_item, aliases, log):
```

Comment:
IIRC I did it with the BBR dataset in mind, where an item can have both a name and a [fastighetsbeteckning](https://github.com/Vesihiisi/COH-tools/blob/master/importer/SeBbrSv.py#L126). I figured that I'd always want the latter as an alias, never as a label -- but relying on addLabelOrAlias seemed a bit tricky, as the dictionary passed to the Uploader could have its keys in whatever order, and thus I wasn't sure I could guarantee that the _actual_ label would be processed first.